docs/service-mesh/10-03-connection-refused.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: Connection refused errors after upgrade
+title: Connection refused errors
 type: Troubleshooting
 ---
 
-Mutual TLS (mTLS) is enabled in the Service Mesh by default. As a result, every element of the Service Mesh must have an Istio sidecar to allow TLS communication. If you don't want to use a sidecar, you can whitelist a service and disable TLS traffic for it by creating a [DestinationRule](https://istio.io/docs/reference/config/networking/destination-rule/).  
+Mutual TLS (mTLS) is enabled in the Service Mesh by default. As a result, every element of the Service Mesh must have an Istio sidecar with a valid TLS certificate to allow communication. Connection between a service with a sidecar and a service without a sidecar results in a `Connection reset by peer` or a `GOAWAY` response. 
 
 - To enable sidecar injection for Pods of existing Services, restart them after upgrading to Kyma 1.0 or higher.
-- To create a DestinationRule for a Service, follow the official [Istio documentation](https://istio.io/docs/reference/config/networking/destination-rule/).
+- To allow connections to a Service with a sidecar from Services without a sidecar, create a [Policy](https://istio.io/docs/reference/config/security/istio.authentication.v1alpha1/) in a `PERMISSIVE` mode.
```

Comment:
```suggestion
- To allow connections between a Service with a sidecar and Services without a sidecar, create an [Authenitcation Policy](https://istio.io/docs/reference/config/security/istio.authentication.v1alpha1/) in the `PERMISSIVE` mode.
```

Reply:
The direction is important in this case.